mavros_msgs/msg/TimesyncStatus.msg

```diff
@@ -0,0 +1,7 @@
+# Status of the MAVLink time synchroniser
+
+std_msgs/Header header
+uint64 remote_timestamp		# remote system timestamp (nanoseconds)
+int64 observed_offset		# raw time offset directly observed from this timesync packet (nanoseconds)
+int64 estimated_offset		# smoothed time offset between companion system and Mavros (nanoseconds)
+float32 round_trip_time		# round trip time of this timesync packet (milliseconds)
```

Comment:
Perhaps good idea to add `_ns/_ms` suffixes.

Reply:
Done!